graph/db/benchmark_test.go

```diff
@@ -763,3 +769,111 @@ func BenchmarkGraphReadMethods(b *testing.B) {
 		}
 	}
 }
+
+// BenchmarkFindOptimalSQLQueryConfig uses the ForEachNode and ForEachChannel
+// methods to find the optimal maximum sqldb QueryConfig values for a given
+// database backend. This is useful for determining the best default values for
+// each backend. The ForEachNode and ForEachChannel methods are used since
+// they make use of both batching and pagination.
+func BenchmarkFindOptimalSQLQueryConfig(b *testing.B) {
+	// NOTE: Set this to true if you want to test with a postgres backend.
+	testPostgres := false
+
+	// NOTE: Set this to true if you want to test with various batch sizes.
+	// By default, page sizes will be tested.
+	testBatching := false
+
+	// Set the various page sizes we want to test.
+	//
+	// NOTE: these are the sqlite paging testing values.
+	testSizes := []int{20, 50, 100, 150, 500}
```

Comment:
Alternatively we could also just binary search for the optimum and not require user changes. Maybe we could even add an lncli command to find these values. wdyt?

Reply:
to add a new command for this here feels a bit out of scope. 

This is just a benchmark to show i i landed at the initial defaults -  it isnt always a clear binary answer, sometimes depending on the run, one may be semi faster than the other. 

defs something we can make more useful later on via a command or something, but i think let's keep the scope here contained for now